scm/build/build-deps.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -ex
+export GOOS=linux
+
+glide install --strip-vendor
+go get -d github.com/vishvananda/netlink
+go get github.com/containernetworking/plugins/pkg/ns
+go get github.com/golang/groupcache/lru
```

Comment:
This file is not needed after #190 
because glide removed and dependencies added to go.mod

Reply:
Agreed. I think that generally there'll be some overlap with #190 , and I actally wonder if it makes sense to merge #190 first and then I'll update this change based on the current situation then?

Reply:
If I were in your shoes, I would do the following:

1. master-master sync between your fork and nokia/danm
2. rebase on your synced fork `master` and step by step resolve every commit conflict
3. cleanup, remove not needed content
4. `git commit --amend` this removes not needed content/files from PR
5. then `git push --force` because of rewriting commit in step 4. by `--amend` ()

For a healthy git history ✌️ 
But maybe you already know what to do. Trust your instincts!😄

Reply:
Yes, thanks. I guess the one thing I hadn't seen, was that #190 did get merged one hour before my comment, and I somehow assumed it was still under review. With it being merged now, I'll go ahead and update this - yes, I've worked with the mechanics of git, thanks :)

Reply:
Done.. and collapsed build-deps/build into a single stage again